frontend/src/pages/Graph/GraphPage.tsx

```diff
@@ -81,6 +81,7 @@ import { canCreate, canUpdate } from '../../types/Permissions';
 import { connectRefresh } from '../../components/Refresh/connectRefresh';
 import { triggerRefresh } from '../../hooks/refresh';
 import { isMultiCluster } from 'config';
+import { ReactNode } from 'react';
```

Comment:
Remove this import and replace `ReactNode` by `React.ReactNode`

Reply:
updated